inc/TRestGeant4Track.h

```diff
@@ -18,26 +18,25 @@
 
 #include <TArrayI.h>
 #include <TColor.h>
+#include <TObject.h>
 #include <TRestGeant4Hits.h>
+#include <TRestGeant4Metadata.h>
+#include <TRestGeant4PhysicsInfo.h>
 #include <TString.h>
 #include <TVector3.h>
 
 #include <iostream>
 #include <vector>
 
-#include "TObject.h"
-
-// Perhaps there might be need for a mother class TRestTrack (if there is future
-// need)
+// Perhaps there might be need for a mother class TRestTrack (if there is future need)
 class TRestGeant4Track : public TObject {
    protected:
-    Int_t fTrack_ID;
-    Int_t fParent_ID;
+    Int_t fTrackID;
+    Int_t fParentID;
```

Comment:
If you change the data member names I believe you must increase the class version --> `ClassDef(TRestGeant4Track, 4);`